src/main/java/gregtech/api/items/materialitem/MaterialMetaItem.java

```diff
@@ -54,6 +54,8 @@ public void registerOreDict() {
             OrePrefix prefix = this.orePrefixes[metaItem / 1000];
             Material material = Material.MATERIAL_REGISTRY.getObjectById(metaItem % 1000);
             OreDictUnifier.registerOre(new ItemStack(this, 1, metaItem), prefix, material);
+            if(prefix.name().equals("dust"))
+                OreDictUnifier.registerOre(new ItemStack(this, 1, metaItem), prefix.name() + "Regular", material);
```

Comment:
I would rather see it like this:
```suggestion

            if(prefix.name().equals("dust"))
                OreDictUnifier.registerOre(new ItemStack(this, 1, metaItem), "dustRegular", material);
```
As it is more obvious on first look